src/libraries/icubmod/embObjStrain/embObjStrain.cpp

```diff
@@ -449,7 +371,7 @@ bool embObjStrain::fillScaleFactor()
     // wait for response
     while(!gotFullScaleValues && (timeout != 0))
     {
-        res->addSetMessage(id32_strain_config, (uint8_t *) &strainConfig);
+        res->setRemoteValue(id32_strain_config, &strainConfig);
         Time::delay(1.0);
```

Comment:
Use `SystemClock::delaySystem()`

Reply:
ok, i have used SystemClock::delaySystem() and SystemClock::nowSystem()